Mac-App/Frontend/Frontend/Views/ExpandableList/DependencyItem.swift

```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
 struct DependencyItem: View {
```

Comment:
I know this is not changed here, but do you think renaming `DependencyItem` to `DependencyItemView` would make sense?

Reply:
Ah nice catch! Yes, I'll do this here 👍

Reply:
https://github.com/Thurman1776-/Uncomplect-us/pull/5/commits/aa98cf3619753885d52cc168664b92adad02bef5